frontend/lib/ui/style/variables.css

```diff
@@ -100,7 +102,10 @@
   --blue-light-800: #065986;
   --blue-dark-100: #d1e0ff;
   --blue-dark-400: #528bff;
-  --menu-updates: #004eeb;
+  --link-default: #155eef;
+  --link-hover: #004eeb;
+  --link-active: #0040c1;
+  --link-visited: #6927da;
```

Comment:
Great, this resolves most of https://github.com/kiesraad/abacus/issues/285 👍 I've ticked of some of the checkboxes there.